### DISCUSSION TITLE
Translations request

### DISCUSSION THREAD

**Discussion by velviagris** [Radiokot/photoprism-android-client#48] [Ideas]
Hi, I like this app after using it, I'd like to help with Chinese translations, is there any plan about it?

**Comment by Radiokot** [comment]
Hi Chen.
Translations are not in priority, but if you want – you can translate the strings and plurals and submit a pull request.
https://github.com/Radiokot/photoprism-android-client/blob/main/app/src/main/res/values/strings.xml
https://github.com/Radiokot/photoprism-android-client/blob/main/app/src/main/res/values/plurals.xml
